src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -914,6 +914,11 @@ private void DynamoView_Loaded(object sender, EventArgs e)
                 // If first run, Collect Info Prompt will appear
                 UsageReportingManager.Instance.CheckIsFirstRun(this, dynamoViewModel.BrandingResourceProvider);
```

Comment:
It's a bit hard to understand from first glance, to me it seems we may want to fold the `dynamoViewModel.HideReportOptions` check into `CheckIsFirstRun()` at https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Services/UsageReportingManager.cs#L154

Reply:
@QilongTang let me give that a shot.